client/src/ContinueIcon.spec.js

```diff
@@ -0,0 +1,9 @@
+import ContinueIcon from "./ContinueIcon";
+import { render } from "@testing-library/react";
+
+describe("<ContinueIcon />", () => {
+  it("renders correctly", () => {
+    const component = render(<ContinueIcon continue />);
+    expect(component.asFragment).toMatchSnapshot();
+  });
+});
```

Comment:
I think if we just want to test that the icon is rendering we can test that in the unit test for the component it's being used inside of, in this case the SubscriptionSecelction component. One idea is to give the continueIcon an `id` or `role` and then finding it by that `id` or `role`.

Reply:
Ok that sounds good to test the continueIcon inside of SubscriptionSelection. I'll add in a test for that. For now I will keep the snapshot test for continueIcon, I could add additional tests here maybe to assert the color prop is working. What do you think?